src/dnaio/_core.pyx

```diff
@@ -62,10 +63,50 @@ cdef class Sequence:
     def __reduce__(self):
         return (Sequence, (self.name, self.sequence, self.qualities))
 
+    def qualities_bytes(self):
```

Comment:
I renamed it `qualities_as_bytes` which is more clear.

On my computer `record.qualities_as_bytes()` only takes half the time of `record.qualities.encode('ascii')` (using the script). This leads to noticable performance differences when bytes are desired:

Using `record.qualities.encode('ascii')`:
```
Benchmark #1: fastq-filter -o /dev/null "median_quality:20" ~/test/big2.fastq
  Time (mean ± σ):      4.686 s ±  0.029 s    [User: 4.504 s, System: 0.181 s]
  Range (min … max):    4.647 s …  4.732 s    10 runs
```

Using `record.qualities_as_bytes()`:
```
Benchmark #1: fastq-filter -o /dev/null "median_quality:20" ~/test/big2.fastq
  Time (mean ± σ):      4.458 s ±  0.037 s    [User: 4.286 s, System: 0.171 s]
  Range (min … max):    4.390 s …  4.497 s    10 runs
```

That is a 5% performance uplift *for the entire application*. That is quite significant.

Given that qualities really only make sense as an array of numbers, `bytes` objects are the best representation of that. `bytes` objects directly translate to the bytes that were read from the file. Adding a method that makes these bytes accessible in the fastest way possible is in scope of the dnaio project if I understood its goals correctly. 

As a side note: while the ascii -> unicode -> ascii convertion has some overhead, I do think it is the most speedy way to check if there are non-ascii (and thus not valid) codes in there.

Reply:
Ah, that was what the first part of your PR description was about. I got confused because this and improving `fastq_bytes` speed are two different things in my opinion (so possibly a separate PR might have been better).

Can you add a comment stating that this is faster than using `record.qualities.encode('ascii')`? Just to clarify why the method exists.

Reply:
Yes, I thought that might have been the case. That was unclear communication on my part sorry.
I will add it to the docstring.